packages/react-native-reanimated/Common/cpp/reanimated/NativeAnimations/NativeAnimationTrace.cpp

```
@@ -0,0 +1,29 @@
+#include <reanimated/NativeAnimations/NativeAnimationTrace.h>
+
+#include <chrono>
+
+namespace reanimated::native_animation {
+
+#ifndef NDEBUG
+
+void recordNativeAnimationTrace(
```

Cppcheck (2.21.0)

[style] The function 'recordNativeAnimationTrace' is never used.

(unusedFunction)